Sources/Utility/StringMangling.swift

```diff
@@ -12,6 +12,11 @@
 /// of them are programming-related.
 extension String {
 
+    /// Returns a form of the string that is a valid bundle identifier
+    public func mangledToBundleIdentifier() -> String {
+        return String(self.mangledToC99ExtendedIdentifier().characters.map { $0 == "_" ? "-" : $0 })
```

Comment:
You're right, it is actually more restrictive and only accepts ASCII characters. I'll update the function.